### ISSUE TITLE
Specify the keyboard traversal focus ordering

### ISSUE THREAD

**Issue by samreid** [phetsims/center-and-variability#257]
Somewhat related to https://github.com/phetsims/center-and-variability/issues/247 and #162 and #256 , we need to specify the keyboard focus order.  We think "kick 1" should be first, for instance.  Some things are less clear, such as where the accordion box should appear in the overall order.

**Comment by catherinecarter** [comment]
Upon first thought, this order seems to make sense for the Variability Screen (after hearing feedback, I'll be able to think about the other screens more easily):

1. Kick 1
2. Kick 5
3. The balls in play area (if there are any)
4. Accordion box open/close
5. Accordion box checkbox
6. Accordion box info button
7. Radio button group: Range
8. Radio button group: Boxplot
9. Radio button group: MAD
10. Checkbox group in play area: Median
11. Checkbox group in play area: Mean
12. Checkbox group in play area: Interval Tool 
13. Player 1
14. Player 2
15. Player 3
16. Player 4
17. Erase
18. Reset All

I'm open to listening to reasons why this list makes sense or doesn't make sense. When I interact with the sim, I usually kick balls, then go back and forth between the accordion box toggles and the measures of spread. That's why I put #4-9 together, but the order I interact with the sim doesn't make sense to make as an order in the sim (bouncing between the radio buttons for measures of spread and the checkboxes).

**Comment by emily-phet** [comment]
I would suggest the following for the Median Screen:

1. Kick 1 button (if present)
2. Kick X button (if present)
3. Soccer balls (if there are any, otherwise skip)
4. Predict Median Checkbox
5. Median checkbox
6. Predict Median slider (if present, otherwise skip)
7. Accordion box toggle
8. Cards (if there are any, skip if none)
9. Sort data checkbox
10. Median checkbox
11. accordion box info dialog button
12. Erase button
13. Reset All button

Note - I believe disappearing buttons (e.g., when Kick 1 or Kick 5 are not available) are not ideal, there is a history around trying to address this issue. @taliesin and @jessegreenberg would know the status of that. I believe a desirable solution was found, but has not been able to implemented. 

For the Mean & Median Screen, I would suggest the same pattern. The only difference that I'm aware of between this screen and the Median screen with respect to controls is that there are more checkboxes by the number line and no cards. Let me know if that is not correct.

For the variability screen, I would suggest the following:

1. Kick 1 button (if present)
2. Kick X button (if present)
3. Soccer balls (if there are any, otherwise skip)
4. Median Checkbox
5. Mean checkbox
6. Interval tool slider - left knob (if present, otherwise skip)
7. Interval tool slider - right knob (if present, otherwise skip)
8. Player radio buttons
9. Graph display radio buttons
10. Accordion box toggle
12. Accordion box checkbox (whatever accordion box you have selected, either the Range, IQR, or MAD checkbox)
14. accordion box info dialog button (whatever accordion box you have selected)
15. Erase button
16. Reset All button

I would suggest both exploring the sim with this alternative input ordering, and also looking at the resulting A11y View to help visualize resulting ordering to help you make decisions on alternative input. 

I believe this sim may benefit from having some customizations in the keyboard help dialog. For example, I think it currently has information re sliders (which is great!) but nothing looks like a slider. It would be nice to have language added that indicated predict and interval tool. It could also be helpful to have information indicated picking up and moving objects (soccer balls and cards).

**Comment by samreid** [comment]
@catherinecarter can you please review the proposal and let us know if it is ready to implement?  Also, let's keep this issue about the keyboard traversal focus order, so I'll move @emily-phet last recommendation about the keyboard help dialog to a new issue.

**Comment by samreid** [comment]
@catherinecarter indicated https://github.com/phetsims/center-and-variability/issues/257#issuecomment-1604609188 seems reasonable and is ready to implement, thanks @emily-phet and @catherinecarter

**Comment by marlitas** [comment]
The pdom order has been set for all three screens. The exact interaction with cards and soccer balls is still being determined( https://github.com/phetsims/center-and-variability/issues/367, https://github.com/phetsims/center-and-variability/issues/351), and I also found a bug that there is no way to close the infoDialogs through keyboard input right now (https://github.com/phetsims/center-and-variability/issues/372). Both of these aspects are being tracked in other issues, and this issue is now ready for design review to confirm pdom order for each screen. 

Over to you @catherinecarter. If all looks good this is also ready for code-review.

**Comment by catherinecarter** [comment]
Tab order looks good on all three screens.

**Comment by samreid** [comment]
The code changes look clear and well-documented. There is some duplication that could be eliminated in the ScreenView pdom orders. For instance, comparing screen 1 and 2:

![image](https://github.com/phetsims/center-and-variability/assets/679486/d7afff34-590f-4298-9298-6176649f721a)

Perhaps we can add an instance method `CAVScreenView.setCenterAndVariabilityPDOMOrder` which allows things like `meanPredictionNode` to be optional (and similar for the variability screen), and sets the order of the specified items.

**Comment by marlitas** [comment]
Met with @catherinecarter and a decision was made to move the kicker button group to be first in the Variability screen.

**Comment by catherinecarter** [comment]
Traversal order looks great. I was able to move between switching kickers, and kicking. It was nice that the tab order took me through all the features of the measures of spread for a single kicker before being prompted to switch kickers. Thanks :)
